tests/ui/errors/remap-path-prefix-reverse.rs

```diff
@@ -1,8 +1,8 @@
 //@ aux-build:remapped_dep.rs
-//@ compile-flags: --remap-path-prefix={{src-base}}/errors/auxiliary=remapped-aux
+//@ compile-flags: --remap-path-prefix={{test-suite-src-base}}/errors/auxiliary=remapped-aux
 
 //@ revisions: local-self remapped-self
-// [local-self] no-remap-src-base: The hack should work regardless of remapping.
+// The hack should work regardless of remapping.
```

Comment:
This is kinda-wrong, in the sense that `no-remap-src-base` is not a directive, but also, because this doesn't work (not `//@`), it ends up functioning as intended.